[
  {
  "id": 1,
  "name": "passesTestOne",
  "class": "test",
  "testsuite": "testsuite-project-name",
  "repository": "test-repo",
  "error": "Multiple error messages have been encountered for this test",
  "test_id": 1,
  "end_date": null,
  "end_reason": null,
  "jira_ticket_url": "https://jira.example.com/browse/TEST-1"
  },
  {
    "id": 2,
    "name": "failsTestFive",
    "class": "test",
    "testsuite": "test",
    "repository": "test-repo",
    "error": "Multiple error messages have been encountered for this test",
    "test_id": 2,
    "end_date": null,
    "end_reason": null
  }
]